### ISSUE TITLE
Hashtags stored with & symbol

### ISSUE THREAD

**Issue by ramyaragupathy** [AmericanRedCross/osm-stats-workers#46]
From @smit1678 -

Certain hashtags are stored with `&` symbol, in the end, while none of the OSM changesets point to these entries:

Examples:
- hotosm-project-5459&
- missingmaps&
- accenture&
- bloombergsingapore&

@smit1678 flagged these entries from the api calls below:

- [https://osm-stats-production-api.azurewebsites.net/group-summaries/bloomber*](https://osm-stats-production-api.azurewebsites.net/group-summaries/bloomber*)
- [https://osm-stats-production-api.azurewebsites.net/group-summaries/hot*](https://osm-stats-production-api.azurewebsites.net/group-summaries/hot*)

Further investigation shows that there are about 148 such hashtags in our backend pointing to 4300+ changesets. _[[Complete List](https://docs.google.com/spreadsheets/d/1r8cJCfOBhDDcoOphlbjLJMj-oWwEii0_ZBqspSuhI3s/edit#gid=129800000)]_

Since the entries are available in our backend, this additional symbol entry must be coming from our worker scripts.

A sample response from an OSM api call. 
```
 <changeset id="63409081" created_at="2018-10-11T09:18:59Z" closed_at="2018-10-11T09:19:04Z" open="false" user="HarmenS" uid="8937686" min_lat="8.7928980" min_lon="5.9959380" max_lat="8.8239297" max_lon="6.0638213" comments_count="0" changes_count="271">
  <tag k="review_requested" v="yes"/>
  <tag k="ideditor:walkthrough_started" v="yes"/>
  <tag k="ideditor:walkthrough_progress" v="welcome;startEditing"/>
  <tag k="changesets_count" v="2"/>
  <tag k="hashtags" v="#hotosm-project-5334;#MissingMaps;#Nigeria;#redcross;#rodekruis;#kadaster20181011"/>
  <tag k="imagery_used" v="Bing aerial imagery;.gpx data file"/>
  <tag k="locale" v="nl"/>
  <tag k="host" v="https://www.openstreetmap.org/edit"/>
  <tag k="created_by" v="iD 2.11.1"/>
  <tag k="comment" v="#hotosm-project-5334&#10;#MissingMaps&#10;#Nigeria&#10;#redcross&#10;#rodekruis&#10;#kadaster20181011"/>
 </changeset>
</osm>
```

Last k-v pair `<tag k="comment" v="#hotosm-project-5334&#10;#MissingMaps&#10;#Nigeria&#10;#redcross&#10;#rodekruis&#10;#kadaster20181011"/>` has all the hashtags separated by a linefeed character.  Probably this is where the [issue arises](https://github.com/AmericanRedCross/osm-stats-workers/blob/master/src/stats.js#L212-L215).

cc @mojodna

**Comment by ramyaragupathy** [comment]
[Hashtag extraction](https://github.com/AmericanRedCross/osm-stats-workers/blob/master/src/stats.js#L212-L215) has a bunch of characters to ignore.

`(#[^\u2000-\u206F\u2E00-\u2E7F\s\\'!"#$%()*,./:;<=>?@[\]^`{|}~]+)` 

 This should include `&` as well

**Comment by mojodna** [comment]
Since `&#10;` is a linefeed character, this suggests that HTML entities should be decoded prior to processing (instead of dropping `&`).

One way to do this is:

```javascript
var htmlEntities = require("html-entities").AllHtmlEntities

htmlEntities.decode("#hotosm-project-5334&#10;#MissingMaps&#10;#Nigeria&#10;#redcross&#10;#rodekruis&#10;#kadaster20181011")
```

**Comment by ramyaragupathy** [comment]
Closed through #47
